ava-test/set.js

```diff
@@ -0,0 +1,80 @@
+import test from 'ava';
+
+import shell from '..';
+import utils from './utils/utils';
+
+const oldConfigSilent = shell.config.silent;
+
+test.beforeEach(t => {
+  t.context.tmp = utils.getTempDir();
+  shell.config.silent = true;
+  shell.cp('-r', 'resources', t.context.tmp);
+});
+
+test.afterEach.always(t => {
+  shell.rm('-rf', t.context.tmp);
+});
+
+const nodeVersion = process.versions.node.split('.').map(str => parseInt(str, 10));
+const uncaughtErrorExitCode = (nodeVersion[0] === 0 && nodeVersion[1] < 11) ? 8 : 1;
```

Comment:
This is some legacy stuff. More of our stuff breaks on v0.10, and even this code doesn't work on v0.8, so I'm just taking it out. The value for v0.11+ is `1`.